streamer/src/nonblocking/quic.rs

```diff
@@ -60,6 +61,9 @@ const CONNECTION_CLOSE_REASON_EXCEED_MAX_STREAM_COUNT: &[u8] = b"exceed_max_stre
 const CONNECTION_CLOSE_CODE_TOO_MANY: u32 = 4;
 const CONNECTION_CLOSE_REASON_TOO_MANY: &[u8] = b"too_many";
 const STREAM_STOP_CODE_THROTTLING: u32 = 15;
+const STREAM_LOAD_EMA_INTERVAL_MS: Duration = Duration::from_millis(5);
+const STREAM_LOAD_EMA_INTERVAL_COUNT: f64 = 10f64;
+const STREAM_LOAD_EMA_SMOOTHING_FACTOR: f64 = 2f64 / (STREAM_LOAD_EMA_INTERVAL_COUNT + 1f64);
```

Comment:
it'd be great to _not_ introduce floating point math here (this commit). fixed point is much easier to reason about

Reply:
Smoothing factor will be `f64`. Are you suggesting to not do the math as part of const statement?

Reply:
no i'm saying not to introduce any `f64` at all. smoothing factor can be specified in integer milli-units or w/e scale makes sense to get the precision we want. do the math in `u128` then scale back up upon output

Reply:
Thanks, that makes sense.